admin/tasks.py

```diff
@@ -5,6 +5,21 @@
 
 HERE = os.path.dirname(os.path.abspath(__file__))
 
+
+@task()
+def manage(*args):
+    """Take arguments and run python manage
+
+    :param args: ex. runserver, migrate
+    """
+    if os.getcwd() != HERE:
+        os.chdir(HERE)
```

Comment:
Because we need to know where manage.py is. I guess I could do a `os.path.join(HERE, '..', 'manage.py')`?